examples/global-parameters-complex.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: global-parameters-complex-
+spec:
+  entrypoint: test-workflow
+  # Parameters can be passed/overridden via the argo CLI.
+  # To override the printed message, run `argo submit` with the -p option:
+  # $ argo submit examples/arguments-parameters.yaml -p message="goodbye world"
```

Comment:
How does the -p option work if the global and local param has the same name, say "message"?

Reply:
I think my understanding is that -p option can only update global params and does not affect local params to any template. It just so happens that global params are also inputs to the entrypoint template but that shouldn't matter for this. So there should be no confusion since you can only update global parameters from command line.